search/search_service/proxy/es_search_proxy.py

```diff
@@ -198,21 +199,24 @@ def _build_filters(self, resource: Resource, filters: List[Filter]) -> List:
         filter_queries: List = []
 
         for filter in filters:
-            filter_name = mapping.get(filter.name) if mapping is not None \
-                and mapping.get(filter.name) is not None else filter.name
+            if mapping is not None and mapping.get(filter.name):
```

Comment:
nit: `if mapping is not None and mapping.get(filter.name) is not None:` for the second check to make sure the filter value doesn't evaluate to false

Reply:
.get evaluates to the value or None and None is false I believe but good call I'll make it consistent